collectors/lib/java.py

```diff
@@ -30,20 +30,19 @@
 
 PERIOD = 60
 
-IGNORED_METRICS = set(["revision", "hdfsUser", "hdfsDate", "hdfsUrl", "date",
-                       "hdfsRevision", "user", "hdfsVersion", "url", "version",
-                       "NamenodeAddress", "Version", "RpcPort", "HttpPort",
-                       # These are useless as-is because they represent the
-                       # thread that's dedicated to serving JMX RPCs.
-                       "CurrentThreadCpuTime", "CurrentThreadUserTime",
-                       # List of directories used by the DataNode.
-                       "StorageInfo",
-                       "VolumeInfo",
-                       "ServerName",
-                       "ZookeeperQuorum",
-                       "Verbose",
-                       "Type"
-                      ])
+IGNORED_METRICS = {"revision", "hdfsUser", "hdfsDate", "hdfsUrl", "date",
```

Comment:
This notation for set creation is relatively new. (https://docs.python.org/2/library/stdtypes.html#set-types-set-frozenset) I worry that older hosts may still be on 2.6 can you sanity check this?